src/entries/popup/pages/home/TokenDetails/TokenContextMenu.tsx

```diff
@@ -52,23 +53,28 @@ export function TokenContextMenu({ children, token }: TokenContextMenuProps) {
       triggerAlert({ text: i18n.t('alert.coming_soon') });
       setSelectedToken(); // clear selected token
     }
-    setClosed(true);
   };
 
   const onSend = () => {
     setSelectedToken(token);
     navigate(ROUTES.SEND);
-    setClosed(true);
   };
 
+  const onBridge = () => {
+    setSelectedToken(token);
+    navigate(ROUTES.BRIDGE);
+  };
+
+  const isBridgeable = token.bridging?.isBridgeable;
+
   const explorer = getTokenBlockExplorer(token);
   const hasExplorerLink = !isNativeAsset(token?.address, token?.chainId);
   const isEth = [token.address, token.mainnetAddress].includes(ETH_ADDRESS);
 
   if (isWatchingWallet && !allowSwap && isEth) return <>{children}</>;
 
   return (
-    <DetailsMenuWrapper closed={closed} onOpenChange={onOpenChange}>
+    <DetailsMenuWrapper closed={true} onOpenChange={onOpenChange}>
```

Comment:
was this on purpose?

Reply:
yes, `closed` is kind of a weird prop but I didn't want to refactor it here, in practice it just removes the `onOpenChange` in the underling `ContextMenu`